### PR TITLE
Fixed the wrong EL APR calculation

### PR DESCRIPTION
**Pull request by Eisei24**

### CODE REVIEW

File: backend/pkg/api/data_access/vdb_management.go [hunk 1]
```diff
@@ -342,10 +342,6 @@ func (d *DataAccessService) GetValidatorDashboardOverview(ctx context.Context, d
 			return nil
 		})
 	}
-	validators, err := d.getDashboardValidators(ctx, dashboardId, nil)
-	if err != nil {
-		return nil, fmt.Errorf("error retrieving validators from dashboard id: %v", err)
-	}
```
Comment (Eisei24) [inline backend/pkg/api/data_access/vdb_management.go:348]: This just looked like a pointless query to have.

File: backend/pkg/api/data_access/vdb_summary.go [hunk 2]
```diff
@@ -871,6 +875,9 @@ func (d *DataAccessService) internal_getElClAPR(ctx context.Context, dashboardId
 	}
 	elIncomeFloat, _ := elIncome.Float64()
 	elAPR = ((elIncomeFloat / float64(aprDivisor)) / (float64(32e18) * float64(rewardsResultTable.ValidatorCount))) * 24.0 * 365.0 * 100.0
+	if math.IsNaN(elAPR) {
+		elAPR = 0
+	}
```
Comment (remoterami) [inline backend/pkg/api/data_access/vdb_summary.go:880]: could log an error here so we can debug, just to be safe

**Review by remoterami — APPROVED** [review]
lgtm

File: backend/pkg/api/data_access/vdb_summary.go (hunk 2 above)
Comment (Eisei24) [reply backend/pkg/api/data_access/vdb_summary.go:880]: I would leave it like it was written for the cl part for now and not add error logging.

**Pull request merged**